components/card/card.css

```diff
@@ -0,0 +1,83 @@
+.card {
+  height: 450px;
+  width: 350px;
+  background-color: white;
+  text-align: center;
+  position: relative;
+}
+
+.card-image {
+  height: auto;
+  width: 100%;
+  border-radius: 0px 0 0 50px;
+  object-fit: cover;
+}
+
+.card-header {
+  margin: 0.2rem auto;
+  padding-top: 0.5rem;
+  font-size: 2rem;
+  color: #474747;
+}
+
+.card-text {
+  display: inline-block;
+  margin: 0.2rem auto;
+  font-size: 1rem;
+  color: #474747;
+}
+
+.card-btn > p {
+  color: #8d8d8d;
+  font-size: 1.5rem;
+  overflow: hidden;
+}
+
+span {
+  display: inline-block;
+  font-size: 1.5rem;
+}
+
+/* textonly card  */
```

Comment:
Since the class name is descriptive enough, you can remove the comment here

Reply:
Sure will change that while refactoring.